sogs/model/room.py

```diff
@@ -1400,15 +1447,15 @@ def get_rooms_with_permission(
     ]
 
 
-def get_readable_rooms(user: Optional[User] = None):
+def get_accessible_rooms(user: Optional[User] = None):
     """
-    Get a list of rooms that a user can access; if user is None then return all publicly readable
+    Get a list of rooms that a user can access; if user is None then return all publicly accessible
     rooms.
     """
     if user is None:
-        result = query("SELECT * FROM rooms WHERE read ORDER BY token")
+        result = query("SELECT * FROM rooms WHERE (read OR accessible) ORDER BY token")
```

Comment:
should this be `read AND accessible` ?

Reply:
I don't think so.  `accessible` only has meaning when `read` is false; when `read` is true we don't care what `accessible` is (or to put it another way: `read` implies a room being accessible).  So a room is accessible to you if: `(read or (not read and accessible))` which simplifies to `read or accessible`.